Lab2/app/src/main/java/nm/lab2/FeedReaderDbHelper.java

```diff
@@ -0,0 +1,101 @@
+package nm.lab2;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+import android.util.Log;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+import nm.lab2.FeedReaderContract.FeedEntry;
+
+/**
+ * Created by nmohamed on 9/22/2015. Class that helps create database for fragments to read from
+ */
+public class FeedReaderDbHelper extends SQLiteOpenHelper {
+    // If you change the database schema, you must increment the database version.
+    public static final int DATABASE_VERSION = 5;
+    public static final String DATABASE_NAME = "FeedReader.db";
+
+    public FeedReaderDbHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+        db.execSQL(FeedEntry.SQL_CREATE_ENTRIES);
+    }
+
+    @Override
+    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        // This database is only a cache for online data, so its upgrade policy is
+        // to simply to discard the data and start over
+        db.execSQL(FeedEntry.SQL_DELETE_ENTRIES);
+        onCreate(db);
+    }
+
+    @Override
+    public void onDowngrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        onUpgrade(db, oldVersion, newVersion);
+    }
+
+    //writes to database
+    public void writeDatabase(String input){
+        SQLiteDatabase dbwrite = getWritableDatabase();
```

Comment:
You can improve performance a bit by declaring this db at the top of the class and then initializing it in the constructor so you only have to call getWritableDatabase once as oppose to every time this method is called.